main.go

```diff
@@ -21,13 +21,13 @@ func main() {
 	err := godotenv.Load(*envFile)
 	if err != nil {
 		log.Fatalf("Error loading .env file at %s. Err: %s\n", *envFile, err)
+		return
```

Comment:
This won't be reachable since the program would have exited already.

Reply:
Erm, did not expect a logging call to terminate the program. I disagree with that approach, but who am I to question the go team :) Fixed.

Reply:
`log.Print*()` functions are what you will end up using most of the time but once in a while, when a non recoverable error has occurred, you can log and exit the program using `log.Fatal*()` functions in one go (pun intended). Typically used when starting http servers, as they are blocking and never return, but when they do, its always an error, so its common to find `log.Fatal(http.ListenAndServe()` calls